docs/visualization/lupton_rgb.rst

```diff
@@ -0,0 +1,26 @@
+**********************************
+Creating color RGB images
+**********************************
+
+`Lupton et al. (2004)`_ describe an "optimal" algorithm for producing red-green-
+blue composite images from three separate high-dynamic range arrays. This method
+is implemented in `astropy.visualization.lupton_rgb` as a set of classes
+providing different scalings and a convenience wraper function. To generate a
+color PNG file with the default (arcsinh) scaling:
+
+>>> from astropy.visualization import lupton_rgb
```

Comment:
I think optional dependencies are not installed on circleci, so you will need to skip all doctest where matplotlib is imported/used. In general use ``.. doctest-skip::`` before the code block.

However, here you don't need the ``>>>`` before the lines, remove them from inside the plot directive, just as it's in the other files (e.g. ``docs/visualization/normalization.rst``, as I realize the wcsaxes one is not yet in this branch)